static/js/jira-configuration.js

```diff
@@ -16,7 +16,7 @@ function openChildWindow(url) {
 
 $(".add-organization-link").click(function(event) {
 	event.preventDefault();
-	window.AP.context.getToken(function(token) {
+	window.AP.context.getToken(function() {
```

Comment:
Should we add context token authentication on that endpoint?

Reply:
@KAbakumovAtl how would you suggest we pass it over to the new url?  This is the issue we had eliminated with link jacking since the JWT token was being passed in the querystring.

Reply:
It is perfectly fine to have JWT token in a query string, because it has expiration window. (It is quite a common practice to place them in query parameters)